Libplanet/Action/UnexpectedlyTerminatedActionException.cs

```diff
@@ -55,6 +58,30 @@ StreamingContext context
         )
             : base(info, context)
         {
+            if (info.TryGetValue(nameof(BlockHash), out byte[] blockHash))
+            {
+                BlockHash = new HashDigest<SHA256>(blockHash);
+            }
+
+            if (info.TryGetValue(nameof(BlockIndex), out long blockIndex))
+            {
+                BlockIndex = blockIndex;
+            }
+
+            if (info.TryGetValue(nameof(TxId), out byte[] txId))
+            {
+                TxId = new TxId(txId);
+            }
+
+            if (info.TryGetValue($"{nameof(Action)}_type", out string actionType))
+            {
+                var values = (Dictionary)new Codec().Decode(
+                    info.GetValue<byte[]>($"{nameof(Action)}_values")
+                );
+
+                Action = (IAction)Activator.CreateInstance(Type.GetType(actionType, true, true));
+                Action.LoadPlainValue(values);
```

Comment:
How about checking `null` here?

Reply:
Did you mean like below?

```suggestion
                Action?.LoadPlainValue(values);
```

Reply:
Yeah, I thought about handling it like that or throwing an exception.